tests/transformers/conftest.py

```diff
@@ -0,0 +1,36 @@
+import gc
+
+import pytest
+from huggingface_hub import scan_cache_dir
+
+
+@pytest.fixture(autouse=True)
```

Comment:
cc @harupy @BenWilson2 is `conftest.py` the recommended way to reuse fixtures across different tests? i can also just copy paste these functions